src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -140,6 +142,26 @@ internal static class CSharpCodeStyleOptions
                 EditorConfigStorageLocation.ForBoolCodeStyleOption("csharp_prefer_simple_default_expression"),
                 new RoamingProfileStorageLocation($"TextEditor.CSharp.Specific.{nameof(PreferSimpleDefaultExpression)}")});
 
+        private static SyntaxKind[] s_preferredModifierOrderDefault =
+            {
+                SyntaxKind.PublicKeyword, SyntaxKind.PrivateKeyword, SyntaxKind.ProtectedKeyword, SyntaxKind.InternalKeyword,
+                SyntaxKind.StaticKeyword,
+                SyntaxKind.ExternKeyword,
+                SyntaxKind.NewKeyword,
+                SyntaxKind.VirtualKeyword, SyntaxKind.AbstractKeyword, SyntaxKind.SealedKeyword, SyntaxKind.OverrideKeyword,
+                SyntaxKind.ReadOnlyKeyword,
+                SyntaxKind.UnsafeKeyword,
+                SyntaxKind.VolatileKeyword,
+                SyntaxKind.AsyncKeyword
+            };
+
+        public static readonly Option<CodeStyleOption<string>> PreferredModifierOrder = new Option<CodeStyleOption<string>>(
```

Comment:
❓ What's stopping this from being `Option<CodeStyleOption<ImmutableArray<string>>>`?